cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -211,19 +212,20 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
         val repositoryConfiguration = actualRepositoryConfigurationFile.takeIf { it.isFile }?.readValueOrNull()
             ?: RepositoryConfiguration()
 
-        val packageCurations = FilePackageCurationProvider.from(packageCurationsFile, packageCurationsDir)
-            .packageCurations.toMutableList()
+        val localCurationProviders = buildList {
+            add(FilePackageCurationProvider.from(packageCurationsFile, packageCurationsDir))
 
-        val repositoryPackageCurations = repositoryConfiguration.curations.packages
+            val repositoryPackageCurations = repositoryConfiguration.curations.packages
 
-        if (config.enableRepositoryPackageCurations) {
-            packageCurations += repositoryPackageCurations
-        } else if (repositoryPackageCurations.isNotEmpty()) {
-            log.warn { "Local package curation were not applied because the feature is not enabled." }
+            if (config.enableRepositoryPackageCurations) {
+                add(SimplePackageCurationProvider(repositoryPackageCurations))
+            } else if (repositoryPackageCurations.isNotEmpty()) {
+                log.warn { "Local package curation were not applied because the feature is not enabled." }
```

Comment:
"curation**s**"
"Local" could be confusing, given that the other provider above is also a "local" provider. Maybe "Package curations from the .ort.yml are not applied...".

Reply:
Please note that I just indented this existing code. So I'll change this in a separate commit.